api/data_explorer/__main__.py

```diff
@@ -65,6 +65,21 @@
 app.add_api('swagger.yaml', base_path=args.path_prefix)
 
```

Comment:
The previous design worked out well because it didn't need the user to specify what the table names were, it just got that information from the list of facets. That had the added benefit of only getting tables that were actually present in the Data Explorer UI.

By de-coupling the two, we're introducing the possibility of exporting tables that don't exist in the Data Explorer facets or forgetting to export tables that do, and putting the burden of correctness on the user. Furthermore, we don't have a clear path towards a smarter export function that links each BQ facet with its data source.

I think it's worth investing into a solution that links facets to their data sources on the back end, which we can use on the front end to do stuff like export to Saturn. What do you think?

Reply:
Good point. I spent some time thinking about this. I think a saved cohort query would look something like:
```
SELECT DISTINCT t1.participant_id
FROM
  (SELECT participant_id
   FROM `project1.dataset1.table1`
   WHERE foo = "FOO") t1
INNER JOIN
  (SELECT participant_id
   FROM `project2.dataset2.table2`
   WHERE bar = "BAR") t2
ON
  t1.participant_id = t2.participant_id
```
Right now, ES index only contains BQ column name ("foo"):
```
{
    foo: "FOO",
}
```
I can change the index to include project/dataset/table:
```
{
    project1.dataset1.table1.foo: "FOO",
}
```
That should give us enough information to construct the query.

This also fixes a problem with the current implementation, where if there 2 tables with field "foo", one would overwrite the other.

Can I do this in a separate PR? I'd have to change the indexer repo first.

Reply:
Or, can you work on this when you get to saving cohort queries?

Just to clarify something which I'm sure you already aware of -- First, just export the whole BQ table with no selection.

Reply:
Let's brainstorm at the next sync. This doesn't affect this PR. The goal of this PR is just to read from `ui.json`, which is unrelated to GCS work. If this needs to change for GCS work, I can change it later.